variables.tf

```diff
@@ -35,7 +35,7 @@ variable "host" {
   description = "NFS server host"
 }
 
-variable "os" {
-  default     = "ubuntu"
-  description = "Server OS that will execute user data script"
+variable "flavor" {
```

Comment:
Not used: https://github.com/cloudposse/terraform-template-user-data-nfs/blob/master/main.tf#L10